src/control/control.c

```diff
@@ -57,6 +57,11 @@ void dt_control_init(dt_control_t *s)
   s->log_message_timeout_id = 0;
   dt_pthread_mutex_init(&(s->log_mutex), NULL);
 
+  s->toast_pos = s->toast_ack = 0;
```

Comment:
You've done a completely different implementation. I was just expecting a new routine, the log_mutex could be reused, no? I don't expect a toast message and a log at the same time anyway? Or maybe this is necessary for a reason I don't see?

Reply:
Well, I'm not sure I want something :) I'm discovering this as we exchange. My idea was to have a minimal set of changes to ensure we could change this later (after discussion) more easily. My initial idea was just to define a dt_loast_lot() using almost the same code but placing the output at a different position and with a smaller timeout. But again I have no strong opinion on all this at this time.

Reply:
The way dt_control_log works I don't think you could easily reposition the output without a separate implementation (or without repositioning the control log too), because the functionality that sets the log text and the functionality that draws it on the screen are separated. I've very much discovered after the fact that what I did might have been a good idea, since I can see the accel toast text potentially overwriting some important error message and I think we do want error text staying on screen.

Reply:
Ok, let's keep this implementation. I have tested it and it is anyway far better than the previous toast as less visible. Please fix the case and I'll merge. Thanks again for all the work!

Reply:
All done and no problem - I'm enjoying myself - I don't get to do nearly enough coding in my day job any more.